packages/dso-toolkit/components/Voorbeelden/10-Patronen/subtitel.njk

```diff
@@ -0,0 +1,28 @@
+{% render '@header', {menu: menu, breadcrumbs: breadcrumbs} %}
```

Comment:
Nou moe, dit is niet hoe we voorbeelden maken.

Use the tool!

Reply:
PS: Waar is de context property `breadcrumbs` ???

Reply:
Ook heb ik een complete refactor gedaan, nu wordt de pagina op een correcte manier met varianten gemaakt.

Reply:
> [..] nu wordt de pagina op een correcte manier met varianten gemaakt.

Echt niet: https://github.com/dso-toolkit/dso-toolkit/blob/93d84639511d1249124624fefdcaf0ad27d47f9f/packages/dso-toolkit/reference/render/subtitel.html

Volgens mij heb je je eigen werk niet bekeken..